paper-tasks-puppeteer/paper-tasks-puppeteer.30m.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env /usr/local/bin/node
+
+
+/******** CONFIGURATION ********/
+const email = '...' // e.g., francesco@buildo.io
+const password = '...' // your dropbox password
+const puppeteerPath = '...' // e.g., /usr/local/lib/node_modules/puppeteer
```

Comment:
Why?

Reply:
bitbar limitation, it needs the exact path to `node` executable and to every global library